catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -165,6 +165,10 @@
 
   private String uiName;
 
+  private Boolean showRelevanceScores = false;
+
+  private Integer relevancePrecision = 5;
```

Comment:
:question: Why are `showRelevanceScores` and `relevancePrecision` declared as Object wrapper types rather than primitive types? Is there a reason that they might be `null`?

Reply:
I was trying to follow the convention of this file... although it does look like some do use the primitive versions

Reply:
I'm leaving the Object versions since most of the other variables use them and primitives get wrapped in their Object versions anyways in the `Map<String,Object>` config map

Reply:
I saw the Slack discussion about problems with Boon with primitives. Fair enough.